docs/source/load-balancing/load-balancing.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Please update the `LoadBalancingPolicy` trait's docstring as well.

Reply:
Done